app/helpers/plate_helper.rb

```diff
@@ -17,6 +18,7 @@ def aliquot_partial
     delegate_missing_to :_presenter
     attr_reader :form, :_presenter
   end
+  # rubocop:enable Rails/HelperInstanceVariable
 
```

Comment:
This just felt tidier, plus it better encapsulates *why* helper instance variables aren't a problem here. (Ie. its not a helper)
I did consider extracting this out into its own file, but it didn't feel quite at hope among the rest of the presenters, due to passing the form object in.